generator/main.py

```diff
@@ -153,10 +153,21 @@ def make_method(record):
 
 def link_structure(struct, types):
     def make_member(m):
-        return StructureMember(Name(m['name']), types[m['type']], m.get('annotation', 'value'))
-
-    # TODO(cwallez@chromium.org): Handle pointer members and their length
-    struct.members = [make_member(m) for m in struct.record['members']]
+        member = StructureMember(Name(m['name']), types[m['type']], m.get('annotation', 'value'))
```

Comment:
nit: member is only used by the return, so maybe we could leave the direct return?

Reply:
oops, must have happened when I was changing things.